.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -626,6 +659,14 @@ dynamodb_GetItem:
             - description:
               snippet_tags:
                 - ruby.example_code.dynamodb.GetItem
+    Swift:
+      versions:
+        - sdk_version: 1
+          github: swift/example_code/ddb/basics

```

Comment:
This should be swift/example_code/ddb -- point to the service folder, not the scenario.